src/main.cpp

```diff
@@ -198,6 +199,23 @@ loop() {
         import_timers(&scheduler);
       }
 
+      // hack to solve openevse bug ignoring charge_current property when going from disabled to active
+      if ((millis() - Timer4) >= 500) {
+        double current = evse.getAmps();
+        double pilot = (double)evse.getChargeCurrent();
+        if (  evse.getAmps() > evse.getChargeCurrent() ) {
+          // Roger we have a problem, reevaluate claims
+          DBUGLN("### Amps > Pilot, reclaim  ###");
+          DBUGVAR(current);
+          DBUGVAR(pilot);
+          evse.evaluateClaims();
+          MicroTask.wakeTask(evse);
+          
+        }
+        Timer4 = millis();
+      }
+  
+
```

Comment:
This really should be hidden away in the EVSE manager/monitor if it is really needed, however getAmps just gets a cached value, so I think ou just need to tag this code into getChargeCurrentAndVoltageFromEvse() which is regularly called